src/style.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+form {
+  position: relative;
+  margin: 20px 0;
+}
+
+input {
+  width: 100%;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: "Avenir", Arial, sans-serif;
+}
+
+input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  box-shadow: none;
+  outline: 0;
+  font-weight: 900;
+  color: #3d4e61;
+}
+
+.big_container::before {
+  content: "";
+  display: block;
+  position: absolute;
+  top: 25.5px;
+  left: 26px;
+  z-index: 1;
+  width: 19px;
+  height: 19px;
+  background-image: url("/images/Search.svg");
```

Comment:
https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md checklist

Reply:
soluted, chek again please

Reply:
*inputs dont inherit font-family from body

Reply:
"font family is usually written for body tag and inherited by others"
input dont inherit font family from body. They are exception. And there are not need to add font family for usual elements like p or div

everything other are fixed